frontend/components/AgentIntroduction/IntroductionStep.tsx

```diff
@@ -122,7 +124,10 @@ export const IntroductionStep = ({
               >
                 <Content title={title} desc={desc} helper={helper} />
               </div>
-              {selectedAgentConfig.isUnderConstruction && <UnderConstruction />}
+              {showUnderConstruction &&
+                selectedAgentConfig.isUnderConstruction && (
+                  <UnderConstruction />
+                )}
```

Comment:
maybe we could have it as an "renderExtraContent()" instead... the component is already too smart, maybe to make it a little more flexible